### PR TITLE
feat(presets): add chat onboarding entry

### PR DESCRIPTION
**Pull request by regischen**

### CODE REVIEW

**Comment by regischen** [comment]
* **#7161** <a href="https://app.graphite.dev/github/pr/toeverything/blocksuite/7161?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @regischen and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/presets/src/ai/provider.ts [hunk 1]
```diff
@@ -84,6 +85,8 @@ export class AIProvider {
     engine: BlockSuitePresets.AIPhotoEngineService
   ): void;
 
+  static provide(id: 'onboarding', fn: (value: boolean) => void): void;
```
Comment (Saul-Mirone) [inline packages/presets/src/ai/provider.ts:88]: Why don't we add `onboarding` to `AIActions`?

Comment (Saul-Mirone) [reply packages/presets/src/ai/provider.ts:88]: @pengx17 @Flrande

**Review by Saul-Mirone — APPROVED** [review]

File: packages/presets/src/ai/provider.ts (hunk 1 above)
Comment (pengx17) [reply packages/presets/src/ai/provider.ts:88]: agreed

**Review by pengx17 — APPROVED** [review]

**Pull request merged**